scrc/utils/test_utils.py

```diff
@@ -0,0 +1,208 @@
+    zh_vg = court_composition_setup(c.ZH_Verwaltungsgericht, ZH_Verwaltungsgericht_test_header, namespace_de)
+    zh_svg = court_composition_setup(c.ZH_Sozialversicherungsgericht, ZH_Sozialversicherungsgericht_test_header, namespace_de)
+
+    test_data = [
+        # ZG_Verwaltungsgericht
+        (zg_vg.president.name, 'Adrian Willimann'),
+        (zg_vg.judges[0].name, 'Adrian Willimann'), 
+        # (zg_vg.judges[1].name, 'Jacqueline Iten-Staub'),
+        # (zg_vg.judges[2].name, 'Matthias Suter'),
+        # (zg_vg.clerks[0].name, 'Patrick Trütsch'),
+        # ZH_Steuerrekurs
+        (zh_sr.president.gender.value, 'male'),
+        (zh_sr.judges[0].name, 'Christian Mäder'),
+        (zh_sr.judges[0].gender.value, 'male'),
+        (zh_sr.judges[1].name, 'Micheline Roth'),
+        (zh_sr.judges[1].gender.value, 'female'),
+        (zh_sr.judges[2].name, 'Barbara Collet'),
+        (zh_sr.judges[2].gender.value, 'female'),
+        (zh_sr.clerks[0].name, 'Hans Heinrich Knüsli'),
+        (zh_sr.clerks[0].gender.value, 'male'),
+        # ZH_Baurekurs
+        (zh_br.judges[0].name, 'Adrian Bergmann'),
+        (zh_br.judges[0].gender.value, 'male'),
+        (zh_br.judges[1].name, 'Stefano Terzi'),
+        (zh_br.judges[1].gender.value, 'male'),
+        (zh_br.judges[2].name, 'Marlen Patt'),
+        (zh_br.judges[2].gender.value, 'male'),
+        (zh_br.clerks[0].name, 'Daniel Schweikert'),
+        (zh_br.clerks[0].gender.value, 'male'),
+        # ZH_Obergericht
+        (zh_og.judges[0].name, 'Th. Kläusli'),
+        (zh_og.judges[0].gender.value, 'male'),
+        (zh_og.judges[1].name, 'K. Vogel'),
+        (zh_og.judges[1].gender.value, 'male'),
+        (zh_og.judges[2].name, 'D. Brugger'),
+        (zh_og.judges[2].gender.value, 'male'),
+        (zh_og.clerks[0].name, 'Ch. Reitze'),
+        (zh_og.clerks[0].gender.value, 'male'),
+        # ZH_Verwaltungsgericht
+        (zh_vg.president.name, 'Tamara Nüssle'),
+        (zh_vg.president.gender.value, 'female'),
+        (zh_vg.judges[0].name, 'Tamara Nüssle'),
+        (zh_vg.judges[0].gender.value, 'female'),
+        (zh_vg.judges[1].name, 'Reto Häggi Furrer'),
+        (zh_vg.judges[1].gender.value, 'male'),
+        (zh_vg.judges[2].name, 'Martin Bertschi'),
+        (zh_vg.judges[2].gender.value, 'male'),
+        (zh_vg.clerks[0].name, 'David Henseler'),
+        (zh_vg.clerks[0].gender.value, 'male'),
+        # ZH_Sozialversicherungsgericht
+        (zh_svg.president.name, 'Mosimann'),
+        (zh_svg.president.gender.value, 'male'),
+        (zh_svg.judges[0].name, 'Mosimann'),
+        (zh_svg.judges[0].gender.value, 'male'),
+        # (zh_svg.judges[1].name, 'Käch'),
+        # (zh_svg.judges[1].gender.value, 'female'),
+        # (zh_svg.judges[2].name, 'Sager'),
+        # (zh_svg.judges[2].gender.value, 'female'),
+        # (zh_svg.clerks[0].name, 'Kudelski'),
+        # (zh_svg.clerks[0].gender.value, 'female'),
```

Comment:
Same here. Maybe add a short comment explaining why they are commented. If they are not relevant, wrong or similar, consider removing them

Reply:
Same as above.